pallets/gear/src/benchmarking/mod.rs

```diff
@@ -341,7 +341,8 @@ benchmarks! {
         let program_id = benchmarking::account::<T::AccountId>("program", 0, 100);
         <T as pallet::Config>::Currency::deposit_creating(&program_id, 100_000_000_000_000_u128.unique_saturated_into());
         let code = benchmarking::generate_wasm2(16.into()).unwrap();
-        benchmarking::set_program(program_id.clone().into_origin(), code, 1.into());
+        let block_number = Pallet::<T>::block_number().unique_saturated_into();
```

Comment:
Some times we use `System::block_number` and some times `pallet_gear_authority::block_number()` can you explain why please?

Reply:
@grishasobol Here we have to use gear block number (the same with pallet_gear_authority::block_number) in functionality code.

I use System::block_number in tests that shouldn't test this functionality cause of several reasons:
 - It's not possible to use pallet gear block_number
 - In such cases it's gives the same result